docs/observability/metrics.md

```diff
@@ -0,0 +1,73 @@
+---
+seotitle: Monitoring your backend application with custom metrics
+seodesc: See how you can monitor your backend application using Prometheus and Encore.
+title: Metrics
+---
+
+Having easy access to useful metrics is a critical part of application observability.
+
+Encore solves this by providing automatic dashboards of all the common application-level
+metrics you need, for each backend service.
+
+Encore also makes it easy to define custom metrics for your application.
+Once defined they automatically show up in the Encore metrics dashboard.
+
+By default Encore exports metrics data to your Cloud Provider's built-in monitoring service.
```

Comment:
Is 'Cloud Provider' capitalized intentionally?

```suggestion
By default Encore exports metrics data to your cloud provider's built-in monitoring service.
```

Reply:
Sounds good